site/src/components/message.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Message
+description: The message component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Fproto%2FamgDNGgGYy2v2Unc6Eet46%2FQ%2526A-Private-Messages%3Fpage-id%3D0%253A1%26type%3Ddesign%26node-id%3D337-533145%26viewport%3D-15969%252C-17033%252C0.75%26scaling%3Dmin-zoom%26starting-point-node-id%3D337%253A548136%26show-proto-sidebar%3D1%26mode%3Ddesign%26t%3DXzNRwbG95jzc2Fvd-1
+classType: Component
+classPrefixes:
+  - message
+---
+
+# Message
+
+Use messages component for chat or messaging communications.
+
+## Usage
+
+Here is an example of a standard usage of messages component.
+
+```html
+<article class="message">
+	<p class="message-time">May 29, 2023, 4:50PM</p>
+	<div class="persona message-persona">
+		<figure class="persona-avatar">
+			<img src="~/src/scaffold/media/avatar.png" alt="Avatar for Jane Doe" />
+		</figure>
+	</div>
+	<div class="message-content">
+		<div class="margin-bottom-xxs">
+			<div class="message-content-header">
+				<p class="message-content-display-name">Q&A Moderator</p>
+				<details class="message-content-options popover popover-right">
```

Comment:
`message-content-options` should stay, but I think any other unrelated classes should probably go. We want a components documentation to be very clean and clear.